### DISCUSSION TITLE
Accidentally transferred USDC into USDC contract address instead of my Kucoin USDC address

### DISCUSSION THREAD

**Discussion by JakeLoong** [blockscout/blockscout#10302] [General]
is there a possibility i can get back my USDC if i accidentally transferred it into USDC contract address instead of my kucoin usdc address? 
this is my transaction https://explorer.immutable.com/tx/0x2206bf1cefd606f3343a29d7425e456124e5fbc28a2750d2abad73bf0ae791a8?tab=index

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
